### DISCUSSION TITLE
Linux kernel 6.2 基本开箱支持了主要硬件

### DISCUSSION THREAD

**Discussion by aqssxlzc** [vrolife/modern_laptop#72] [General]
更新之后把wmi,mic,5582be这些驱动都卸载了，目前还正常

**Comment by vrolife** [comment]
WMI 和键盘背光还是有用的。这两个驱动目前并不完美，所以我也没什么心思并入主线。

**Comment by pekumbe** [comment]
Hello!
Thanks for the drivers! They helped a lot!
I don't know if this is an appropriate place to comment on that... The integrated webcam on my Redmi Note Pro 14 (2022) is not working on kernel 6.2.7-2-MANJARO.
Does it need a specific driver as well?

**Reply by vrolife** [reply]
Generally, webcam is a normal USB device that does not require special drivers, and can be confirmed through command 'lsusb'.
On my laptop:
```
x:~$ lsusb |grep -i webcam
Bus 004 Device 002: ID 2b7e:b557 SunplusIT Inc XiaoMi USB 2.0 Webcam
x:~$ uname -a
Linux x 6.2.7-060207-generic #202303170542 SMP PREEMPT_DYNAMIC Fri Mar 17 13:17:41 UTC 2023 x86_64 x86_64 x86_64 GNU/Linux
```

Test the camera:
```
ffplay /dev/video0
```
